### ISSUE TITLE
No output

### ISSUE THREAD

**Issue by tennyson-mccalla** [lackhove/m4baker#6]
After pressing "process all" the percentage under "chapter #" shoots right up to 100% immediately and there's no output.

Changing output directories makes no difference.

What's going on?

**Comment by themightyoarfish** [comment]
Same here. Seems broken on Ubuntu 18.04.

**Comment by FMorsbach** [comment]
I had the same issue and looked a bit into the problem. Toggling the `--verbose` option from the command line shows a problem with how sox is called and a look into the source code reveals that it can be fixed easily. A quick test shows that this is the fix. I was about to make a pull request when I saw that there is already a PR for the same issue from 2015 (see #3) . As there is no response in 5 years now, maybe its time for a fork?

**Comment by tennyson-mccalla** [comment]
@FMorsbach , thanks for the solution. I’m going to implement this ASAP. I think a fork is definitely appropriate here as @crabmanX might not be with us anymore.

**Comment by tennyson-mccalla** [comment]
UPDATE: @FMorsbach, still nothing. Just jumps right to 100% with no output. The baseclasses.py file in the src folder have been fixed though. I suppose that wasn't the issue.

It seems that there isn't a single m4b making tool anywhere on the internet that works for linux. A shame.

**Comment by FMorsbach** [comment]
Did you also install it again? So running `python setup.py install --optimize=1` ? 
Because it does work for me.

**Comment by tennyson-mccalla** [comment]
Yes, completely deleted the original and reinstalled with the pull request. I checked the baseclasses.py files to be sure that it was different (it was).

Nothing doing.

**Comment by jrad232** [comment]
Thanks for this thread. I had this same issue. I was able to use Sonic-Y3k's fork and got it to work for me.

**Comment by tennyson-mccalla** [comment]
I finally did get it to work for me after I used the hint about the `—verbose ` mode. That made me realize that it didn’t work with files saved with a variety of sample rates. So I just made it all uniform and it ran fine.
